docs/basic-config.asciidoc

```diff
@@ -255,6 +255,10 @@ const client = new Client({
 |`boolean`, `'proto'`, `'constructor'` - By the default the client will protect you against prototype poisoning attacks. Read https://web.archive.org/web/20200319091159/https://hueniverse.com/square-brackets-are-the-enemy-ff5b9fd8a3e8?gi=184a27ee2a08[this article] to learn more. If needed you can disable prototype poisoning protection entirely or one of the two checks. Read the `secure-json-parse` https://github.com/fastify/secure-json-parse[documentation] to learn more. +
 _Default:_ `false`
 
+|`caFingerprint`
+|`string` - If configured, verify the supplied certificate authority fingerprint. +
```

Comment:
__optional nit:__ maybe that's just me, but it sounds a bit confusing, and I'd rather say (since CA is a body/organization that doesn't have a fingerprint :slightly_smiling_face: ):
```suggestion
|`string` - If configured, verify the supplied CA certificate fingerprint. +
```
It has become very common to call CA certificate as just CA though, so up you you.

Reply:
I would also add a sentence that the fingerprint is SHA-256 digest in HEX format, maybe with an example